src/mlpack/methods/reinforcement_learning/environment/continuous_mountain_car.hpp

```diff
@@ -140,7 +169,9 @@ class ContinuousMountainCar
     double force = math::ClampRange(action.action[0], -1.0, 1.0);
 
     // Update states.
-    nextState.Velocity() = state.Velocity() + force * duration - 0.0025 *
+    nextState.Velocity() = state.Velocity() + force * duration -
+            //0.0025 *
+            0.0014*//for the simulation to be theoretically able to reach the end goal, this value must be less than the value of duration
```

Comment:
If we're hard coding this value, does it make sense to have a comment like this?